commit f31703528863d6748631c953426a75450f250850
Author: Hawking Zhang <Hawking.Zhang@amd.com>
Date:   Sun Sep 8 09:09:15 2019 +0800

    drm/amdgpu: enable error injection to XGMI block via debugfs
    
    allow inject error to XGMI block via debugfs node ras_ctrl
    
    Signed-off-by: Hawking Zhang <Hawking.Zhang@amd.com>
    Reviewed-by: Guchun Chen <guchun.chen@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.c
index f3cbe4dc4748..994d2b34094c 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.c
@@ -704,6 +704,7 @@ int amdgpu_ras_error_inject(struct amdgpu_device *adev,
 		break;
 	case AMDGPU_RAS_BLOCK__UMC:
 	case AMDGPU_RAS_BLOCK__MMHUB:
+	case AMDGPU_RAS_BLOCK__XGMI_WAFL:
 		ret = psp_ras_trigger_error(&adev->psp, &block_info);
 		break;
 	default: